packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -10,6 +10,46 @@ $ds-c-inset-border-width: $spacer-half;
   opacity: 0;
   position: absolute;
 
+  + label {
```

Comment:
Tried to also start organizing this SCSS file into `.ds-c-choice` and `.ds-c-choice--inverse` sections

Reply:
Hmm, this whole stylesheet is pretty confusing in general... I was thinking of separating the blocks into the 3 major variants, `ds-c-choice`, `ds-c-choice--inverse` and `ds-c-choice--error`. `ds-c-choice__checkedChild` would be another block altogether. Would you mind if i gave a shot at organizing it later?

Reply:
That's fine by me, thanks for asking!

Reply:
I take back what I said, what you laid out here is really good. I made some minor changes but kept most of what you organized here